.idea/.gitignore

```diff
@@ -0,0 +1,3 @@
+# Default ignored files
+/shelf/
```

Comment:
there are 179 changed files . please remove all generated files and re-open the PR